openff/nagl/features/_base.py

```diff
@@ -42,7 +42,24 @@ def __init__(cls, name, bases, namespace, **kwargs):
 
 
 class Feature(ImmutableModel, abc.ABC):
+    """
+    Abstract base class for atom and bond features.
+
+    Features with length one can simply inherit :py:class:`AtomFeature
+    <openff.nagl.features.atoms.AtomFeature>` or
+    :py:class:`BondFeature <openff.nagl.features.bonds.BondFeature>`,
+    implement :py:class:`_encode <encode>`, and define
```

Comment:
Ah cool, I thought this had to be something like 
```
:py:func:`Feature._encode`
```
but magically `:py:class: <encode>` knows where to go!

Reply:
Yeah sometimes Sphinx magic makes things harder but sometimes it's good :)